site/src/scss/_base.scss

```diff
@@ -48,4 +61,7 @@ pre code {
       line-height: rem-calc(22);
     }
   }
+  &.blue {
+    @extend %blue;
```

Comment:
No need to have this in a placeholder.  We just did this in dashing because we needed it in .dashing and .dashing-icon.

Reply:
OK cool - I swapped this to just a class.